src/amo/reducers/currentView.js

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+import { CURRENT_VIEW_SET } from 'core/constants';
+
+
+export type currentViewTypes = {|
+  // TODO: Create an addonType Type that is one of type (ADDON_TYPE_EXTENSION,
+  // ADDON_TYPE_THEME, etc.)
+  addonType: string | null,
+  isExploring: boolean | void,
+  isHomePage: boolean | void,
```

Comment:
When I look at where `isHomePage: true` is dispatched, it seems to be a duplication of `isExploring` since that's the home page. Can you just say `const isHomePage = isExploring` ? Or is there a future scenario in the desktop mocks that I'm missing?